src/Foundation/ErrorServiceProvider.php

```diff
@@ -57,6 +63,15 @@ public function register()
             ];
         });
 
+        $this->container->singleton('flarum.error.contents', function () {
+            return [
+                NotAuthenticatedException::class => NotAuthenticated::class,
```

Comment:
Should these be codes or status strings? That might be cleaner for exceptions added by extensions.